tests/testthat/test-export.R

```diff
@@ -8,6 +8,9 @@ testthat::test_that("Listing print correctly", {
 })
 
 testthat::test_that("key columns repeat with export_as_txt", {
+  require("dplyr", quietly = TRUE)
+  skip_if_not_installed("dplyr")
+
```

Comment:
it's better to skip first and then do something if not skipped already

Reply:
```suggestion
  skip_if_not_installed("dplyr")
  require("dplyr", quietly = TRUE)
```